src/classes/CRLP_RollupProcessor.cls

```diff
@@ -499,8 +508,20 @@ public inherited sharing class CRLP_RollupProcessor {
         SObject updatedRecord = handler.getPopulatedSObject();
 
         // Determine if the updated SObject record is different than the parent.
-        // If not, set it to null
-        if (CRLP_Rollup_SVC.resultsNeedUpdate(parent, updatedRecord, rollupDefs) == false) {
+        Boolean needsUpdate = CRLP_Rollup_SVC.resultsNeedUpdate(parent, updatedRecord, rollupDefs);
+
+        // Unconditionally set force skew flag if parent is Account/Contact and running in skew mode
+        if (mode == CRLP_RollupProcessingOptions.BatchJobMode.SkewMode && hasSkewFlag) {
```

Comment:
I think if you check for isForceSkewModeSet being false here, then the rest of the code in the if block only runs when the field needs to be set to true, which also means that an update is required.

```suggestion
        if (mode == CRLP_RollupProcessingOptions.BatchJobMode.SkewMode && hasSkewFlag && !isForceSkewModeSet) {
```